tools/wptrunner/wptrunner/tests/test_products.py

```diff
@@ -18,7 +19,9 @@ def test_load_active_product(product):
     # test passes if it doesn't throw
 
 
-@all_products("product")
+@all_products("product", marks={
+    "firefox": pytest.mark.xfail(sys.version[0] == "3", reason="mozinfo 0.10 doesn't support py3"),
```

Comment:
mozinfo 1.0.0 shipped six months ago, why does @pyup-bot not realise it exists?!

Reply:
I've emailed PyUp support about this; in the mean time, try bumping the release manually and see if we can avoid this xfail?

Reply:
I'd prefer doing that when we're actually running this test in CI